### ISSUE TITLE
Nintendo NES emulator: NesNes

### ISSUE THREAD

**Issue by IbeVanmeenen** [fcambus/jsemu#61]
Suggestion:
NESNES (New EcmaScript NES), a pure JS NES emulator by @koenkivits.
https://github.com/koenkivits/nesnes

**Comment by BigEd** [comment]
/sub
One for @fcambus to decide, I think. It would be nice to have a demo to link to as well as the source.
/cc @koenkivits

**Comment by koenkivits** [comment]
Another project I made (x-nes, an NES web component) does have a demo page (http://koen.kivits.com/x-nes/) and it has NESNES running under the hood. Still, I'd also be up for creating a separate demo page for NESNES if it helps. :)

(Thanks for mentioning my project @IbeVanmeenen!)

**Comment by BigEd** [comment]
Please do @koenkivits! I tried your x-nes demo and nothing happened - Firefox and Chrome on OSX, Chrome on Linux. Safari said it couldn't do it but the others just sit there with a grey screen. Is it user error?

**Comment by fcambus** [comment]
@koenkivits a demo page would be great indeed, please go ahead :)

Once this is done, the emulator can be added to the list, feel free to open a PR. Thanks!

**Comment by koenkivits** [comment]
Then I'll create a demo page. :)

@BigEd: ouch, apparently it was my own fault. Not sure how I didn't notice it before, but it should be fixed now!
